source/js/components/newsletter-signup/organisms/form-specific-style.js

```diff
@@ -47,15 +47,25 @@ export const FORM_STYLE = {
     buttonStyle: "secondary",
     buttonWidthClasses: "tw-w-auto",
     fieldStyle: "outlined",
-    formContainerClass: "tw-flex-col tw-w-1/2 large:tw-pl-24 tw-mt-8",
+    formContainerClass: "tw-w-1/2 large:tw-pl-24 tw-mt-8",
     headingLevel: 3,
     headingClass: "tw-h1-heading",
     innerWrapperClass: "tw-flex tw-gap-8",
-    introContainerClass: "tw-flex-col tw-w-1/2 large:tw-pr-24",
+    introContainerClass: "tw-w-1/2 large:tw-pr-24",
   },
   footer: {
     fieldStyle: "outlined",
     headingLevel: 4,
     headingClass: `tw-h5-heading`,
   },
+  "header:2-col": {
+    buttonStyle: "primary",
+    buttonWidthClasses: "tw-w-full",
+    fieldStyle: "filled",
+    formContainerClass: "tw-w-full large:tw-w-1/2",
+    headingLevel: 2,
+    headingClass: "tw-h5-heading",
+    innerWrapperClass: "tw-flex tw-flex-col large:tw-gap-16 large:tw-flex-row",
```

Comment:
The original form implementation has a 30px "gap" (set with Bootstrap). I went with `large:tw-gap-16` (32px) as we don't have a 30px equivalent preset class.